pkg/manifests/nmstateconfig.go

```diff
@@ -0,0 +1,68 @@
+package manifests
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	"github.com/openshift/assisted-service/models"
+	"github.com/pkg/errors"
+)
+
+func getNMStateConfig() aiv1beta1.NMStateConfig {
+	var nmStateConfig aiv1beta1.NMStateConfig
+	if err := GetFileData("nmstateconfig.yaml", &nmStateConfig); err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
```

Comment:
I realized after that this probably shouldn't exit if the file isn't found, since the nmStateConfig file is optional.

Reply:
We expect at least one nmstateconfig to be provided which will help to define the node0